client/src/features/rootV2/NavbarV2.tsx

```diff
@@ -158,10 +158,7 @@ export default function NavbarV2() {
 
   return (
     <>
-      <header
-        className={cx("bg-navy", "navbar-expand-lg")}
-        data-bs-theme="dark"
-      >
+      <header className="navbar-expand-lg" data-bs-theme="navy">
```

Comment:
To make the `color` and `bg` work:

```suggestion
      <header
        className={cx("navbar-expand-lg", "text-body", "bg-body")}
        data-bs-theme="navy"
      >
```

Reply:
Ha, thanks!
I initially tried with the variables for text color and background color but it didn't work because of this. And I went for a quicker shortcut instead... :roll_eyes: